openapi/platform/platform.yaml

```diff
@@ -2133,14 +2133,31 @@ components:
         relationships:
           type: object
           required:
-            - businessLocation
             - automation
           properties:
-            businessLocation:
+            automation:
               type: object
-              description: A link to the business location that this automation is being run for.
+              description: 'A link to the automation definition that is being run. '
               required:
                 - data
+              properties:
+                data:
+                  type: object
+                  required:
+                    - type
+                    - id
+                  properties:
+                    type:
+                      type: string
+                      default: automations
+                      example: automations
+                    id:
+                      type: string
+                      example: Automation-91cb048c-c498-43ba-ab99-2ca2d27d2c46
+                      pattern: ^Automation-
+            businessLocation:
+              type: object
+              description: A link to the business location that this automation is being run for. Either the businessLocation or order is required. (It depends on the actions that the automation uses.)
```

Comment:
Hm, it doesn't really depend on the actions the automation uses. It depends on the "automation type". Maybe should update that here and on the order one below to be more clear.

Reply:
Please go ahead and update the wording.